app/uk/gov/hmrc/agentclientrelationships/connectors/DesConnector.scala

```diff
@@ -191,21 +183,20 @@ class DesConnector @Inject()(httpClient: HttpClient, metrics: Metrics, agentCach
     val from: String = LocalDate.now().minusDays(showInactiveRelationshipsDays).toString
     val regime = "AGSV"
     val url = new URL(
-      s"$desBaseUrlFeatureSwitched/registration/relationship?arn=$encodedAgentId&agent=true&active-only=false&regime=$regime&from=$from&to=$now")
+      s"$desBaseUrl/registration/relationship?arn=$encodedAgentId&agent=true&active-only=false&regime=$regime&from=$from&to=$now")
 
     val cacheKey = s"${arn.value}-$now"
     agentCacheProvider.agentTrackPageCache(cacheKey) {
-      getWithDesHeaders(s"GetInactiveRelationships", url, desAuthTokenFeatureSwitched, desEnvFeatureSwitched).map {
-        response =>
-          response.status match {
-            case Status.OK =>
-              response.json.as[InactiveRelationshipResponse].relationship.filter(isNotActive)
-            case Status.BAD_REQUEST                             => Seq.empty
-            case Status.NOT_FOUND                               => Seq.empty
-            case _ if response.body.contains("AGENT_SUSPENDED") => Seq.empty
-            case other: Int =>
-              throw UpstreamErrorResponse(response.body, other, other)
-          }
+      getWithDesHeaders(s"GetInactiveRelationships", url, desAuthToken, desEnv).map { response =>
+        response.status match {
+          case Status.OK =>
+            response.json.as[InactiveRelationshipResponse].relationship.filter(isNotActive)
+          case Status.BAD_REQUEST                             => Seq.empty
+          case Status.NOT_FOUND                               => Seq.empty
+          case _ if response.body.contains("AGENT_SUSPENDED") => Seq.empty
+          case other: Int =>
+            throw UpstreamErrorResponse(response.body, other, other)
+        }
```

Comment:
The three match blocks (lines 120, 141, 190) could all be factored out into one.

Reply:
yeah...these have always been this way...wanted to keep existing untouched for time being...once we get things working we can look at code refactoring

Reply:
:+1: